patches/minecraft/net/minecraft/entity/player/ServerPlayerEntity.java.patch

```diff
@@ -130,23 +130,15 @@
     }
  
     protected void func_70670_a(EffectInstance p_70670_1_) {
-@@ -1139,6 +1160,7 @@
-    }
- 
-    public void func_71033_a(GameType p_71033_1_) {
-+      if (!net.minecraftforge.common.ForgeHooks.onChangeGameMode(this, this.field_71134_c.func_73081_b(), p_71033_1_)) return;
```

Comment:
Not sure why this line is being removed?

Reply:
I'm not sure why that happened either, I didn't touch that